procinterrupts.c

```diff
@@ -57,7 +57,17 @@ static int check_platform_device(char *name, struct irq_info *info)
 	DIR *dirfd;
 	char path[512];
 	struct dirent *ent;
-	int rc = -ENOENT;
+	int rc = -ENOENT, i;
+	static struct pdev_irq_info{
```

Comment:
Space after pdev_irq_info and {

Reply:
concur with @ppwaskie , fix the space error and we can pull this